apache/superset
# chore: Homepage cleanup

### SUMMARY
This is follow up to https://github.com/apache/superset/pull/14695. This pr address the comments and ensures that the the sticky tabs works when examples tabs is rendered and not viewed.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
load a brand new instance of superset and test. The examples tabs should be set first if there is no active tab set in local state  or viewed is set in previous state.

### ADDITIONAL INFORMATION
